sql/2019/09_Accessibility/09_01_b.sql

```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# 09_01_b: % of pages having any heading
```

Comment:
Not a big deal but the convention I've been using has been more like `09_01b.sql`.